.eslintrc.json

```diff
@@ -276,7 +293,18 @@
       ],
       "parser": "@typescript-eslint/parser",
       "rules": {
-        "unused-imports/no-unused-imports": "error"
+        "unused-imports/no-unused-imports": "error",
+        "@typescript-eslint/no-empty-object-type": "off",
+        "@typescript-eslint/no-unsafe-declaration-merging": "off",
+        "no-duplicate-imports": "off",
+        "@typescript-eslint/consistent-type-imports": [
```

Comment:
inline type imports were [added in TS 4.5](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names).  Since we technically "support" TS 4.4, we need use "separate-type-imports" instead of "inline-type-imports" so that our types are parsable on TS 4.4.